website/source/docs/platform/k8s/out-of-cluster-nodes.html.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: "docs"
+page_title: "Out-of-Cluster Nodes - Kubernetes"
+sidebar_current: "docs-platform-k8s-ooc-nodes"
+description: |-
+  Non-Kubernetes nodes can join a Consul cluster running within Kubernetes. These are considered "out-of-cluster" nodes.
+---
+
+# Out-of-Cluster Nodes
+
+Non-Kubernetes nodes can join a Consul cluster running within Kubernetes.
+These are considered "out-of-cluster" nodes.
```

Comment:
Bikeshedish but "out-of-cluster" might be confusing since it's meaning "out-of-kubernetes" but is talked about in the same sentence as "Consul cluster" which is is _not_ out of!

What do you think about calling these "Non-Kubernetes nodes" or similar. It's kinda gross I admit but less ambiguous?

Reply:
Yeah, "out-of-cluster" is the term Kubernetes uses but not frequently enough that we need to stick to it, let me rename.

Reply:
Actually, I want to keep "out-of-cluster" because it COULD be Kubernetes nodes that are also in a different Kubernetes cluster. I define what "out-of-cluster" means right in that first sentence, but I'll add another sentence to clarify that it does not mean "out-of-Consul-cluster."

Does that work?